src/test_l1_dsHdmiIn.c

```diff
@@ -2254,10 +2316,10 @@ void test_l1_dsHdmiIn_negative_dsGetEdid2AllmSupport(void) {
     UT_ASSERT_EQUAL(dsHdmiInInit(), dsERR_NONE);
 
     // Step 3: Call dsGetEdid2AllmSupport() with invalid inputs
-    DS_ASSERT_AUTO_TERM_NUMERICAL(dsGetEdid2AllmSupport(dsHDMI_IN_PORT_MAX, &allmSupport), dsERR_INVALID_PARAM);
+    UT_ASSERT_EQUAL(dsGetEdid2AllmSupport(dsHDMI_IN_PORT_MAX, &allmSupport), dsERR_INVALID_PARAM);
 
     // Step 4: Call dsGetEdid2AllmSupport() with invalid inputs
-    DS_ASSERT_AUTO_TERM_NUMERICAL(dsGetEdid2AllmSupport(dsHDMI_IN_PORT_0, NULL), dsERR_INVALID_PARAM);
+    UT_ASSERT_EQUAL(dsGetEdid2AllmSupport(dsHDMI_IN_PORT_0, NULL), dsERR_INVALID_PARAM);
 
     // Step 5: Terminate the HDMI input using dsHdmiInTerm()
     UT_ASSERT_EQUAL(dsHdmiInTerm(), dsERR_NONE);
```

Comment:
Why CHECK_FOR_EXTENDED_ERROR_CODE is not for the dsERR_NOT_INITIALIZED enhanced case?

Reply:
done